schema/meltano.schema.json

```diff
@@ -27,6 +27,67 @@
       "description": "A database URI for the project system database. Defaults to a SQLite file stored at .meltano/meltano.db",
       "default": "sqlite:///$MELTANO_PROJECT_ROOT/.meltano/meltano.db"
     },
+      "state_backend": {
+        "type": "object",
+          "description": "Configuration for state backend.",
+          "properties": {
+            "uri": {
+              "type": "string",
+              "description": "The URI pointing to the backend to use.",
+              "default": "systemdb"
+              },
+            "lock_timeout_seconds": {
+              "type": "integer",
+              "description": "The number of seconds a lock on a state file is considered valid."
+              },
+            "lock_retry_seconds": {
+              "type": "integer",
+              "description": "The number of seconds to wait between retrying lock acquisition."
+              },
+            "az": {
+              "type": "object",
+              "description": "Configuration for Azure Blob Storage state backend.",
+              "properties": {
+                "connection_string": {
+                  "type": "string",
+                  "description": "The azure connection string to use for connecting to Azure Blob Storage."
+                  }
+                }
+            },
+          "gs": {
```

Comment:
I've seen "gcs" used as an acronym for this, but not "gs". Is there precedence for this?

Reply:
Good callout. Yeah, the URI schema for google cloud storage is `gs` but for the settings I think it makes sense to use `gcs` for clarity and consistency with the `extras`. Updated.